components/hooks/useNotes.ts

```diff
@@ -62,33 +62,30 @@ export const useNotes = ({ userId }: UseNotesProps) => {
   }, [userId]);
 
   const updateNote = useCallback(
-    async (note: {
-      id: string;
-      title: string;
-      content: string;
-      public?: boolean;
-    }) => {
+    async (
+      note: {
+        id: string;
+        title: string;
+        content: string;
+        public?: boolean;
+      },
+      publish?: boolean
+    ) => {
       if (!userId || !note) return;
 
       const notesRef = doc(db, "users", userId, "notes", note.id);
-      const publicRef = doc(db, "public_notes", note.id);
       const currentTime = new Date().getTime();
-
-      const publicNote = await getDoc(publicRef);
+      const updatedContent = publish
+        ? { ...note, updatedAt: currentTime, publishedAt: currentTime }
+        : { ...note, updatedAt: currentTime };
```

Comment:
```suggestion
      const updatedContent = !public
        ? { ...note, updatedAt: currentTime, publishedAt: currentTime }
        : { ...note, updatedAt: currentTime };
```

Reply:
Won't it be `note.public`?

Also, I should've done it in the first place. It was unnecessary to modify the updateNotes call